pkg/util/util.go

```diff
@@ -53,6 +54,12 @@ func ContextWithCancelOnSignals(ctx context.Context, sigs ...os.Signal) context.
 		for {
 			s := <-sig
 			cancels++
+			reporter.Get().ReportStep(reporter.CliStepData{
```

Comment:
actually, now that I think about it, there is no way from watching the events received, to connect them into a single installation attempt. This means it would hard to connect back the dots and see that a specific installation attempt was ended in a cancellation. 
To solve it I would generate inside the `Init` method of the analytics reporter a uid that would be reported for each event and called `flowId`. later this id can be used to trace back a whole flow, or installation attempt.